packages/cli/create-strapi-starter/src/utils/prompt-user.ts

```diff
@@ -9,7 +9,7 @@ interface Answers {
 
 // Prompts the user with required questions to create the project and return the answers
 export default async function promptUser(projectName: string, starter: string, program: Program) {
-  const questions: inquirer.QuestionCollection = [
+  const questions: ReadonlyArray<inquirer.DistinctQuestion<Answers>> = [
```

Comment:
`QuestionCollection` can be an observable, but looking at how it's used i've changed it to be tighter instead of changing the variable name :)